src/org/zaproxy/zap/extension/websocket/files/scripts/templates/websocketsender/WebsocketSender Default Template.js

```diff
@@ -0,0 +1,12 @@
+// Note that new WebSocketSender scripts will initially be disabled
```

Comment:
Just noticed the file is missing in `ZapAddOn.xml` file

Reply:
I have it, just forgot to add it